src/types/TableMetrics.ts

```diff
@@ -0,0 +1,32 @@
+import { ajv } from '../ajv';
+import type { JSONSchemaType } from 'ajv';
+
+export interface TableMetrics {
+  count: number;
+  maxId: number;
+  now: Date;
+}
+
+export const tableMetricsSchema: JSONSchemaType<TableMetrics> = {
+  type: 'object',
+  properties: {
+    count: {
+      type: 'integer',
+    },
+    maxId: {
+      type: 'integer',
+    },
+    now: {
```

Comment:
If we were to include it in the response, we should use [`timestamp`](https://www.hl7.org/fhir/bundle-definitions.html#Bundle.timestamp) rather than `now`. But we're not currently:

https://github.com/PhilanthropyDataCommons/service/blob/84bc24afedf0383bf9cfff76faa1a463033f2013/src/database/accessors/loadBundle.ts#L19-L22

That said, the argument in favor of including the timestamp in the response rather than just the headers is to allow clients to more easily store responses offline and process them later - including humans who just want to `curl > proposals.json` and then later open that in their editor. This is an important use case in FHIR, but I don't know how important it is to us!

Reply:
If we were ever to include it in the response I would be fine with the `timestamp` name, but would rather it mean something more like "the datetime when the total count was estimated or found" or the "last modified" of the contents of the bundle rather than the FHIR "bundle assembled" definition. If callers can use an API, surely they can get the current datetime locally and/or read the `Date` header in the server response. For example, I similarly would object to including the HTTP status in the body. A [response object should have it all](https://axios-http.com/docs/res_schema), usually, right?

Anyway, not an issue at the moment, as @jasonaowen pointed out, because we aren't including it, thankfully!

Reply:
Yeah, to be clear -- the response body currently is just using part of the table metrics type (but table metrics is used in testing and the original implementation had it so I maintained it in the refactor)

I would advocate for us to *not* include timestamp in any part of the response right now just in the sense that it isn't necessary to accomplish the intended scope of this PR and there is not an immediate use for it.